src/Tree.tsx

```
@@ -350,7 +350,9 @@ class Tree<TreeDataType extends DataNode | BasicDataNode = DataNode> extends Rea
     };
 
     function needSync(name: string) {
-      return (!prevProps && name in props) || (prevProps && prevProps[name] !== props[name]);
+      return (
+        (!prevProps && props.hasOwnProperty(name)) || (prevProps && prevProps[name] !== props[name])
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)